content/operate/rs/references/compatibility/client-side-caching.md

```diff
@@ -0,0 +1,84 @@
+---
+Title: Client-side caching compatibility with Redis Software and Redis Cloud
+alwaysopen: false
+categories:
+- docs
+- operate
+- rs
+description: Redis Software and Redis Cloud compatibility with client-side caching.
+linkTitle: Client-side caching
+toc: 'true'
+weight: 80
+---
+
+Redis Software and Redis Cloud support [client-side caching]({{<relref "/develop/connect/clients/client-side-caching">}}) for databases with Redis versions 7.4 or later.
+
+## Required database versions
+
+Client-side caching in Redis Software and Redis Cloud requires Redis database versions 7.4 or later.
+
+The following table shows the differences in client-side caching support by product:
+
+| Redis product           | Client-side caching support |
+|-------------------------|-----------------------------|
+| Redis Community Edition | Redis v6.0 and later |
+| Redis Cloud             | Redis database v7.4 and later |
+| Redis Software          | Redis database v7.4 and later |
+
+## Supported RESP versions
+
+Client-side caching in Redis Software and Redis Cloud requires RESP3.
```

Comment:
Not sure if this is really necessary but thought I'd suggest it. (The linked page might be too technical anyway.)
```suggestion
Client-side caching in Redis Software and Redis Cloud requires [RESP3]({{< relref "/develop/reference/protocol-spec#resp-versions" >}}).
```